sample/home.jsx

```diff
@@ -151,9 +151,9 @@ export default class Home extends Component {
             this.presel = presel;
```

Comment:
Shouldn't we remove the "ref trick" in the example now that it's not needed anymore to extract values ?